gnome-initial-setup/pages/live-chooser/eos-setup-live-user

```diff
@@ -37,34 +37,59 @@ EOS_INSTALLER = 'eos-installer.desktop'
 GDM_APPS_DIR = '/usr/share/gdm/greeter/applications'
 EOS_INSTALLER_PATH = os.path.join(GDM_APPS_DIR, EOS_INSTALLER)
 
-SCHEMA = 'org.gnome.shell'
+SHELL_SCHEMA = 'org.gnome.shell'
 ICON_GRID_LAYOUT = 'icon-grid-layout'
 TASKBAR_PINS = 'taskbar-pins'
 
+GS_SCHEMA = 'org.gnome.software'
+ALLOW_UPDATES = 'allow-updates'
+
+class SettingsUpdates(object):
```

Comment:
I would rebase this entire commit 'eos-setup-live-user: Disallow updates on GNOME Software' into the previous one that added this script. Unlike upstream projects where it's nicer to have more and smaller commits, for us it's nicer to have fewer commits to backport. ;)